test/framework/integration_framework/integration_test_framework.cpp

```diff
@@ -375,6 +383,21 @@ namespace integration_framework {
     return *this;
   }
 
+  IntegrationTestFramework &IntegrationTestFramework::checkStatus(
+      const shared_model::interface::types::HashType &tx_hash,
+      std::function<void(const shared_model::proto::TransactionResponse &)>
```

Comment:
`checkProposal`, `checkVerifiedProposal`, `checkBlock` use interface type for lambdas. Please use it here as well for consistency: 
`std::function<void(const TxResponseType &)>`